ansible/roles/docker/tasks/compose-install-v1.yml

```diff
@@ -9,11 +9,14 @@
 - name: Download the Docker Compose V1 executable
   ansible.builtin.get_url:
     url: "{{ docker_compose_v1_url }}"
-    dest: "/usr/bin/docker-compose"
+    dest: "/usr/local/bin/docker-compose"
     owner: root
     group: root
     mode: 0755
     checksum: "sha256:{{ lookup('url', docker_compose_v1_digest_url) | regex_search('[0-9a-f]{64}') }}"
+  environment:
+    http_proxy: "{{ http_proxy }}"
+    https_proxy: "{{ http_proxy }}"
```

Comment:
I tend to think proxy env vars should be set at the playbook level rather than directly in roles. If you do want to add them in the role, then consider adding a `| default(omit)` to make the `http_proxy` variable optional. We have a similar thing in site.yml:
```
environment: "{{ proxy_env | default({}) }}"
```